.github/workflows/update-ami-ids.yaml

```diff
@@ -42,8 +42,8 @@ jobs:
       - name: Assume AWS role
         uses: aws-actions/configure-aws-credentials@010d0da01d0b5a38af31e9c3470dbfdabdecca3a # v4.0.1
         with:
-          aws-region:  us-west-2
-          role-to-assume: "arn:aws:iam::126027368216:role/tf-teleport-ami-gha-role"
+          aws-region: us-west-2
+          role-to-assume: "arn:aws:iam::146628656107:role/tf-teleport-ami-gha-role"
```

Comment:
Should this move to an env var in github-terraform instead? Seems kinda odd to hard code a role ARN here.

Reply:
I have no objections to that. I am just fixing what is currently here. :-)